### PR TITLE
add vs project guid in compilation input telemetry

### PR DESCRIPTION
**Pull request by heejaechang**

### CODE REVIEW

**Comment by heejaechang** [comment]
@KevinH-MS can you take a look?

File: src/VisualStudio/Core/Def/Telemetry/ProjectTelemetryIncrementalAnalyzerProvider.cs [hunk 1]
```diff
@@ -161,8 +164,12 @@ public Task AnalyzeProjectAsync(Project project, bool semanticsChanged, Invocati
                 {
                     try
                     {
+                        var workspace = (VisualStudioWorkspaceImpl)project.Solution.Workspace;
+                        var vsProject = workspace.GetHostProject(projectId);
```
Comment (KevinH-MS) [inline src/VisualStudio/Core/Def/Telemetry/ProjectTelemetryIncrementalAnalyzerProvider.cs]: Can this ever return null?

**Comment by KevinH-MS** [comment]
LGTM...this is so we can cross-reference/join with other event streams that already use ProjectGuid as an identifier?

**Comment by heejaechang** [comment]
@KevinH-MS yes.

**Pull request merged**